integration/spark2/src/main/scala/org/apache/spark/sql/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
+
+
+__all__ = [
```

Comment:
I think it is better to add example and doc to describe how to use it in the same PR